### PR TITLE
Create blog post for v1.36.0

### PR DESCRIPTION
**Pull request by DNin01**
Announces every new notable feature in v1.36.0.
 
This is the first time I think we've ever embedded a video inside Markdown syntax on our website, so if someone could test this, that'd be great. Also, does anyone know if videos will work properly in our RSS feed? (Do we still use our RSS feed? I've never checked it out.)

I noted in this blog post that this update would be _rolling out soon_. I think most of the recent major updates have actually rolled out gradually[^1], but I just thought it'd be a good idea to note for those who are confused about why they haven't received the update yet. It also means we can announce this update a little earlier, maybe like a day after we upload it to the Chrome Web Store for review?

[^1]: > And the Chrome Web Store has only released to 60% of users. I didn't want to release at everyone at once.
    
    https://github.com/ScratchAddons/ScratchAddons/issues/6062#issuecomment-1551449262

### CODE REVIEW

**Comment by Samq64** [comment]
> This is the first time I think we've ever embedded a video inside Markdown syntax on our website, so if someone could test this, that'd be great. Also, does anyone know if videos will work properly in our RSS feed? (Do we still use our RSS feed? I've never checked it out.)

I am subscribed to the RSS feed but I'm not sure how to test it before it goes live.

**Comment by WorldLanguages** [comment]
~~no one uses rss~~

File: content/blog/v1-36-released.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+---
+title: What's new in Scratch Addons v1.36
+description: Rearrange custom block parameters, play with new customization options, and more!
+date: null
```
Comment (WorldLanguages) [inline content/blog/v1-36-released.md:4]: reminder

**Comment by DNin01** [comment]
The videos are not playing.

<sub>Windows 11 23H2, Edge 121</sub>

Looks like they're `` tags instead of `<video>` tags. They work if I inspect element and replace the code with:
```html
<video src="/assets/img/blog/v1-36-released/reordering-parameters.mp4" alt="Custom block parameters being rearranged in the “Make a Block” modal" type="video/webm" controls loop></video>
```

**Comment by BroJac5246** [comment]
They're working for me, Chrome 114 on ChromeOS 114

**Comment by BroJac5246** [comment]
Oh NVM it's because you fixed it

**Pull request merged**